packages/@aws-cdk/aws-ses/README.md

```diff
@@ -112,3 +112,91 @@ new ses.AllowListReceiptFilter(this, 'AllowList', {
 ```
 
 This will first create a block all filter and then create allow filters for the listed ip addresses.
+
+## Email sending
+
+### Dedicated IP pools
+
+When you create a new Amazon SES account, your emails are sent from IP addresses that are shared with other
+Amazon SES users. For [an additional monthly charge](https://aws.amazon.com/ses/pricing/), you can lease
+dedicated IP addresses that are reserved for your exclusive use.
+
+Use the `DedicatedIpPool` construct to create a pool of dedicated IP addresses:
+
+```ts
+new ses.DedicatedIpPool(this, 'Pool');
+```
+
+### Configuration sets
+
+Configuration sets are groups of rules that you can apply to your verified identities. A verified identity is
+a domain, subdomain, or email address you use to send email through Amazon SES. When you apply a configuration
+set to an email, all of the rules in that configuration set are applied to the email.
+
+Use the `ConfigurationSet` construct to create a configuration set:
+
+```ts
+declare const myPool: ses.IDedicatedIpPool;
+
+new ses.ConfigurationSet(this, 'ConfigurationSet', {
+  customTrackingRedirectDomain: 'track.cdk.dev',
+  suppressionReasons: ses.SuppressionReasons.COMPLAINTS_ONLY,
+  tlsPolicy: ses.ConfigurationSetTlsPolicy.REQUIRE,
+  dedicatedIpPool: myPool,
+});
+```
+
+### Email identity
```

Comment:
Just a question, do you know if CloudFormation waits for the verification or does it just submit
the request and return without waiting? I guess the question is can you create an `EmailIdentity`
without the identity ever getting verified?

Reply:
CF doesn't wait, so yes you can create an identity that remains unverified. This allows to run integ tests 🙂